src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -4,24 +4,69 @@ import CardBody from './CardBody';
 
 const PreviewCard = ({ concept, closeModal, addConcept }) => {
   const {
-    display_name, descriptions, mappings, display_locale, id, url,
+    display_name,
+    descriptions,
+    mappings,
+    display_locale,
+    id,
+    url,
+    version,
+    created_on,
+    type,
+    concept_class,
+    datatype,
+    source,
+    owner,
+    owner_type,
```

Comment:
@Tittoh did linting come into play here?

Reply:
@AGMETEOR , I think it had to because he also had to add more props there.